man/systemd.network.xml

```diff
@@ -1691,6 +1691,16 @@
           </listitem>
         </varlistentry>
 
+        <varlistentry>
+          <term><varname>SendMUDURL=</varname></term>
+          <listitem>
+            <para>When configured, the Manufacturer Usage Descriptions (MUD) URL will be sent to the DHCPV6 server.
+            Takes an URL in the range 1 to 255 characters. DHCPv6 clients are intended to have at most one
```

Comment:
Takes an URL of length up to 255 characters.

I'd also add: A superficial verification that the string is a valid URL will be performed.